libs/remix-lib/src/execution/txRunnerVM.ts

```diff
@@ -74,7 +75,7 @@ export class TxRunnerVM {
     }
   }
 
-  runInVm (from: string, to: string, data: string, value: string, gasLimit: number, useCall: boolean, callback: VMExecutionCallBack) {
+  async runInVm (from: string, to: string, data: string, value: string, gasLimit: number, useCall: boolean, callback: VMExecutionCallBack) {
```

Comment:
why `async` ?

Reply:
below I am doing `await this.getVMObject().vm.evm.journal.checkpoint()`.